app/src/main/res/layout/fragment_home.xml

```diff
@@ -48,81 +48,77 @@
+        app:layout_constraintBottom_toBottomOf="@id/bottom_bar"
+        app:layout_constraintEnd_toStartOf="@+id/add_tab_button"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent">
+        app:layout_constraintTop_toTopOf="@id/bottom_bar">
 
-        <org.mozilla.fenix.home.SearchView
-            android:id="@+id/toolbar_wrapper"
-            android:transitionName="toolbar_wrapper_transition"
-            android:layout_width="0dp"
-            android:layout_height="40dp"
-            android:layout_marginEnd="0dp"
+        <ImageView
+            android:id="@+id/search_engine_icon"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_gravity="start|center_vertical"
             android:layout_marginStart="8dp"
-            android:layout_marginTop="8dp"
-            android:layout_marginBottom="8dp"
-            android:background="@drawable/home_search_background_normal"
-            android:clickable="true"
-            android:contentDescription="@string/search_hint"
-            android:focusable="true"
-            app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toStartOf="@+id/add_tab_button"
-            app:layout_constraintStart_toStartOf="parent">
-
-            <ImageView
-                android:id="@+id/search_engine_icon"
-                android:layout_width="24dp"
-                android:layout_height="24dp"
-                android:layout_marginStart="8dp"
-                android:layout_gravity="start|center_vertical"
-                android:clickable="false"
-                android:focusable="false"
-                android:importantForAccessibility="no" />
+            android:clickable="false"
+            android:focusable="false"
+            android:importantForAccessibility="no" />
 
-            <TextView
-                android:id="@+id/toolbar"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="start|center_vertical"
-                android:layout_marginStart="44dp"
-                android:clickable="false"
-                android:focusable="false"
-                android:lines="1"
-                android:ellipsize="end"
-                android:importantForAccessibility="no"
-                android:text="@string/search_hint"
-                android:textColor="?primaryText"
-                android:textSize="15sp" />
-        </org.mozilla.fenix.home.SearchView>
+        <TextView
+            android:id="@+id/toolbar"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="start|center_vertical"
+            android:layout_marginStart="44dp"
+            android:clickable="false"
+            android:ellipsize="end"
```

Comment:
No action needed but for future reference: I notice the order of a few of these attributes changed. It's preferable to keep them the same when making changes like this because the diff tools can optimize better and better retain blame.